### ISSUE TITLE
Often crash on open terminal

### ISSUE THREAD

**Issue by opravil-jan** [curusarn/resh#172]
Hi,

very often when I open terminal I see the message shown below:
error message
`# RESH SEARCH APP failed - sorry for the inconvinience - check '/home/opravil/.cache/resh/cli_last_run_out.txt' and '~/.resh/cli.log' `

OS: debian bullseye

`# cat /home/xxx/.cache/resh/cli_last_run_out.txt / /does not show anything`

`# cat .resh/cli.log
2022/03/12 14:10:36 resh-daemon is not running - try restarting this terminal
2022/03/27 10:40:05 resh-daemon is not running - try restarting this terminal
2022/03/27 10:40:18 resh-daemon is not running - try restarting this terminal
2022/03/27 10:40:32 resh-daemon is not running - try restarting this terminal
2022/03/27 21:15:04 resh-daemon is not running - try restarting this terminal
2022/03/27 21:15:28 resh-daemon is not running - try restarting this terminal
2022/03/30 07:57:11 resh-daemon is not running - try restarting this terminal
2022/03/31 05:36:28 resh-daemon is not running - try restarting this terminal
2022/03/31 05:36:52 resh-daemon is not running - try restarting this terminal
2022/03/31 22:09:13 resh-daemon is not running - try restarting this terminal
2022/04/01 10:12:23 resh-daemon is not running - try restarting this terminal`

Thanks 
John

**Comment by opravil-jan** [comment]
Hi, 
any response?

**Comment by tivvit** [comment]
Hi John, are you using the latest version (run `reshctl update`)?

Is this happening after the reboot or during normal workflow?

**Comment by opravil-jan** [comment]
Yes I'm. I made it work by deleting whole history and started from scratch,.
